convert

```diff
@@ -1,6 +1,9 @@
 # To convert a file from jpg to pdf
 convert original.jpg converted.pdf
 
+# To rotate an image to 180 degrees
+convert original.jpg -rotate 180 converted.jpg
```

Comment:
```suggestion
convert <path/to/original.jpg> -rotate 180 <path/to/converted.jpg>
```

Maybe it's better and more obvious where arbitrary values can be placed.

Reply:
Thank you for your suggestion, it is a good idea.